tests/test_usm_utils.py

```diff
@@ -0,0 +1,42 @@
+import dpctl.tensor as dpt
+from numpy.testing import assert_equal
+
+import dpnp
+
+
+def test_byte_bounds():
+    a = dpt.usm_ndarray((3, 4), dtype="int64", buffer="shared")
+    values = dpt.arange(12, dtype="int64")
+    for i in range(3):
+        a[i, :] = values[i * 4 : (i + 1) * 4]
+    low, high = dpnp.byte_bounds(a)
+    assert_equal(high - low, a.size * a.itemsize)

```

Comment:
It's preferable to use `assert` directly for comparing scalar values:
```suggestion
    assert (high - low) == (a.size * a.itemsize)
```